### DISCUSSION TITLE
Utilizar logo de la licencia AGPL de en vez de texto

### DISCUSSION THREAD

**Discussion by ianvatega** [PucelaBits/websegura#135] [Ideas]
Tipo de licencia al final de la web de en vez del texto poner el logo http://www.gnu.org/graphics/license-logos.html

**Comment by adrm** [comment]
¿Cuál es la ventaja del logo? Creo que es un cambio innecesario. Quien sepa qué es va a saberlo con el texto, y quien no lo sepa le va a dar igual que sea un logo o solo un texto.

**Reply by palmerabollo** [reply]
Por mí no, tiene casi 1 año ya y es la versión con "Active LTS".

**Reply by adrm** [reply]
Supongo que esto es para el hilo de node 12 vs node 14 :P

**Reply by palmerabollo** [reply]
Sí, no sé que ha pasado :D Aprovecho para opinar sobre el logo, creo que recarga la página y las personas con dificultades visuales (ej. un ciego) tendrán más problemas con la imagen que con el texto.

**Comment by ianvatega** [comment]
Ok, cierro el tema.
